### PR TITLE
BZ-1855261: Updating to approve pending kubelet serving CSRs

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1855261

Preview: https://deploy-preview-34975--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-3-expired-certs.html#dr-scenario-3-recovering-expired-certs_dr-recovering-expired-certs

### CODE REVIEW

File: modules/dr-recover-expired-control-plane-certs.adoc [hunk 1]
```diff
@@ -5,11 +5,11 @@
 [id="dr-scenario-3-recovering-expired-certs_{context}"]
 = Recovering from expired control plane certificates
 
-As of {product-title} 4.4.8, the cluster can automatically recover from expired control plane certificates. You no longer need to perform the manual steps that were required in previous versions.
+The cluster can automatically recover from expired control plane certificates.
 
-The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates.
+However, you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates. For user-provisioned installations, you might also need to approve pending kubelet serving CSRs as well.
```
Comment (jeana-redhat) [inline modules/dr-recover-expired-control-plane-certs.adoc:10]: maybe rm either `also` or `as well`

File: modules/dr-recover-expired-control-plane-certs.adoc [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
 
-Use the following steps to approve the pending `node-bootstrapper` CSRs.
+Use the following steps to approve the pending CSRs.
```
Comment (jeana-redhat) [inline modules/dr-recover-expired-control-plane-certs.adoc:12]:
```suggestion
Use the following steps to approve the pending CSRs:
```
per ISG

**Review by jeana-redhat — COMMENTED** [review]
little nits, LGTM

**Comment by bergerhoffer** [comment]
@xingxingxia Can you please QE review this (or let me know if there is someone else to ask), thanks!

Preview: https://deploy-preview-34975--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-3-expired-certs.html#dr-scenario-3-recovering-expired-certs_dr-recovering-expired-certs

**Comment by xingxingxia** [comment]
/lgtm
Thanks

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**